egs/wsj/s5/steps/cleanup/combine_short_segments.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Copyright 2015  Tom Ko
+# Apache 2.0
+
+# Begin configuration section.
+frame_shift=10
+# End configuration section.
+
+echo "$0 $@"
+
+[ -f ./path.sh ] && . ./path.sh
+. parse_options.sh || exit 1;
+
+if [ $# -ne 3 ]; then
+  echo "This script concatenates segments with length shorter than the specified length in seconds."
+  echo ""
+  echo "Usage: steps/cleanup/combine_short_segments.sh [options] <minimum-length> <data-in> <data-out>"
```

Comment:
Please change  <code> <data-in> <data-out></code>  to <code> <input-data-dir> <output-data-dir></code>